assets/sass/protocol/includes/forms/_lib.scss

```diff
@@ -29,11 +29,13 @@ $field-padding: $spacing-sm;
 $field-h-spacing: $spacing-md;
 $field-v-spacing: $spacing-lg;
 $field-focus-ring: 0 0 0 2px rgba($color-blue-40, 0.5);
-$field-focus-ring-dark: 0 0 0 2px color.mix($color-blue-40, $color-white);
+$field-focus-ring-dark: 0 0 0 2px $color-white;
+$field-focus-ring-outline: 2px solid #afccf3;
```

Comment:
I added `#afccf3` instead of using `rgba($color-blue-50, 0.3)` because the set opacity was basically invisible in dark theme, so I got the hexcode of the lighter color and placed it in the code instead. Happy to hear of other suggestions.